src/liquid/components/ld-badge/readme.md

```diff
@@ -36,6 +40,15 @@ permalink: components/ld-badge/
   </svg>
 </ld-badge>
 
+<!-- React component -->
+
+<LdBadge>
+  <svg {...{ slot: 'icon' }} viewBox="0 0 24 24" fill="none">
```

Comment:
Why not just slot="icon"? 🤷‍♂️

Reply:
> Why not just slot="icon"? 🤷‍♂️

Typescript would complain. It doesn't know slot. 😣

Reply:
Damn, expected that answer. 😂
Is there any way to extend the React type? 🤔

Reply:
This seems to be an issue with our sandbox app. The type is missing on the svg element. If you use a span, it's there. So it's somehow available on HTMLAttributes but missing on SVGProps. I don't think it's an issue with our library.